src/plugins/interactive_setup/server/elasticsearch_service.ts

```diff
@@ -324,6 +326,35 @@ export class ElasticsearchService {
       }
     }
 
+    // This check is a security requirement - Do not remove it!
+    this.logger.debug(`Verifying that host "${host}" responds with Elastic product header`);
+
+    // Using native Client for this check instead of Kibana's `elasticsearch.createClient()` wrapper
+    // since it throws `NoLivingConnectionsError` when sending an OPTIONS request. We can't use a
+    // GET request either since we don't have valid credentials yet and the product header isn't
+    // returned for unauthenticated requests.
+    const verifyClient = new Client({
+      node: host,
+      ssl: {
+        rejectUnauthorized: false,
+      },
+    });
```

Comment:
I don't think this is necessary, it looks like the problem is you're closing `client` above on line 309:
https://github.com/elastic/kibana/blob/dbbe51bf569db8a7ba68facadd09d16464509d1b/src/plugins/interactive_setup/server/elasticsearch_service.ts#L309
when I removed that and changed the OPTIONS below to use `client` instead, I was able to complete the interactive setup without running into a `NoLivingConnectionsError`.

Reply:
Ahhhh, nice, let me try that!